modules/gitops-argocd-cli-global-options.adoc

```diff
@@ -0,0 +1,122 @@
+// Module is included in the following assemblies:
+//
+// * gitops_cli_argocd/argocd-gitops-cli-reference.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="gitops-argocd-cli-global-options_{context}"]
+= Global options
+
+Global options are applicable to all the subcommands of `argocd`.
+
+.Global options
+[options="header",cols="1,1,1"]
+|===
+|Option |Argument type |Description
+
+|`--auth-token`
+|`string`
+|Authentication token.
+
+|`--client-crt`
+|`string`
+|Client certificate file.
+
+|`--client-crt-key`
+|`string`
+|Client certificate key file.
+
+|`--config`
+|`string`
+|Path to Argo CD configuration file. Defaults to `/home/user/.config/argocd/config` path.
+
+|`--controller-name`
+|`string`
+|Name of the Argo CD Application Controller component with `argocd-application-controller` as the default label. 
+
+If the label of this component's name differs from the default, for example, when you are installing it through the Helm chart, set either the `--controller-name` option or the `ARGOCD_APPLICATION_CONTROLLER_NAME` environment variable to specify the name of the component. 
+
+|`--core`
+|N/A
+|If set to `true`, the CLI communicates directly with the Kubernetes API server instead of using the Argo CD API server.
+
+|`--grpc-web`
+|N/A
+|Enable the gRPC-Web protocol for the Argo CD server, for example, if the server is behind a proxy that does not support the HTTP/2 protocol.
+
+|`--grpc-web-root-path`
+|`string`
+|Enable the gRPC-Web protocol for the Argo CD server, for example, if the server is behind a proxy that does not support the HTTP/2 protocol. Set web root.
+
+|`-H, --header`
+|`string`
+|Configure an additional header to all requests made by the {gitops-shortname} `argocd` CLI. You can add multiple headers by setting this option multiple times. This option also supports comma-separated headers.
+
+|`-h, --help`
+|N/A
+|Help for {gitops-shortname} `argocd` CLI.
+
+|`--http-retry-max`
+|`integer`
+|Set the maximum number of retries to establish an HTTP connection to the Argo CD server.
+
+|`--insecure`
+|N/A
+|Skip server certificate and domain verification.
+
+|`--kube-context`
+|`string`
+|Direct the command to the given kube context. 
+
+|`--logformat`
+|`string`
+|Set the logging format to either text or JSON. Defaults to `text`.
+
+|`--loglevel`
+|`string`
+|Set the logging level. Defaults to `info`. The `debug`, `info`, `warn`, and `error` levels are available.
```

Comment:
It might be worth noting that the levels are listed in order of verbosity.

Reply:
we can mention that the levels are listed in the decreasing order of verbosity.